tests/virtctl/ssh.go

```diff
@@ -86,4 +98,33 @@ var _ = Describe("[sig-compute][virtctl]SSH", decorators.SigCompute, func() {
 		Entry("using the native ssh method", cmdNative),
 		Entry("using the local ssh method", cmdLocal),
 	)
+
+	It("local-ssh flag should be unavailable in virtctl binary", func() {
+		if nativeSSHAvailable {
+			Skip("virtctl was built with native ssh golang support")
+		}
+
+		_, cmd, err := clientcmd.CreateCommandWithNS(util.NamespaceTestDefault, "virtctl", "ssh", "--local-ssh=false")
+		Expect(err).ToNot(HaveOccurred())
+
+		out, err := cmd.CombinedOutput()
+		Expect(err).To(HaveOccurred(), "out[%s]", string(out))
+		Expect(string(out)).To(Equal("unknown flag: --local-ssh\n"))
+	})
 })
+
+func isNativeSSHAvailable() bool {
```

Comment:
The preferred way would be to introduce a tag for the native SSH. This will allow us to focus or skip these tests and express what test we want to actually execute. For example, we can prevent a mistake when the client is compiled without the flag even if the intention was to include it.

Reply:
@xpivarc do you mean a go build tag? It was the previous version of this PR, but I've suggested simplifying it

Reply:
That was indeed the previous version of this PR. I'm fine with both, just need to know which one is preferred.

Reply:
No, I mean `Serial` or [labels](https://onsi.github.io/ginkgo/#spec-labels)

Reply:
Unfortunately it is not possible to have a label filter and programmatic focus at the same time. Certain tests need to be excluded by default and by filtering them with labels the programmatic focus is disabled. IMO this breaks the usual developer workflow which is bad.

See https://github.com/onsi/ginkgo/blob/729bd402e6de48bcf70cbf7ab485d743e9b210ea/internal/focus.go#L72